src/sonic-host-services/scripts/hostcfgd

```diff
@@ -56,6 +73,7 @@ def run_cmd(cmd, log_err = True):
         if log_err:
             syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}"
                   .format(err.cmd, err.returncode, err.output))
+    return 0
```

Comment:
It should be the subprocess return code and err.returncode in case of exception.

Reply:
1) This is still not fixed. It would still return 0, even in case of exception
2) Noticed there is a duplicate run_cmd at line 40